vision/product-search/cloud-client/src/main/java/vision/snippets/PurgeProducts.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vision.snippets;
```

Comment:
Keeping the same package as the others is fine: `package com.example.vision;`

Reply:
Can I keep it as it is?

Reply:
Better to be consistent with the rest of the files.